py_ecc/fields/field_elements.py

```diff
@@ -254,8 +270,10 @@ def __rmul__(self: T_FQP, other: Union[int, T_FQP]) -> T_FQP:
         return self * other
 
     def __div__(self: T_FQP, other: Union[int, T_FQP]) -> T_FQP:
-        if isinstance(other, int_types_or_FQ):
-            return type(self)([c / other for c in self.coeffs])
+        if isinstance(other, int):
```

Comment:
Is it possible that `other` should include the `FQ` type in the Union? Not sure if this function was ever taking an argument of that type, do you have more context there?

Reply:
You're right, there's a conflict with `other` type, `if` options, and the error message. Will work more on this.

Reply:
Digging back through blames, we can see the application of types here:
https://github.com/ethereum/py_ecc/commit/0309225c56acdb081f535ff6353fff2cad4f8329#diff-93c798286fdea4df041291e658b99539423804a846045a425261d05b2e0e6691

My best guess is that the dropping of the `FQ` option was a typo. We can see `__mul__`, `__rmul__`, `__div__` and `__truediv__` all got types updated to use the generics, but only `__mul__` kept `int, T_FQ`, `T_FQP`, with no logic change in any of the functions. This doesn't make sense, at least for `__rmul__` because it's passing `other` over to `__mul__` which still says it accepts `T_FQ`.

So yeah, I think `FQ` was dropped in error. Adding it back.